docs/source/managed-federation/uplink.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Apollo Uplink
+sidebar_title: Uplink
+---
+
+When using [managed federation](./overview/), your federated gateway regularly polls an endpoint called **Apollo Uplink** for its latest supergraph schema and other configuration:
+
+```mermaid
+graph LR;
+  subgraph "Your infrastructure"
+  serviceA[Products subgraph];
+  serviceB[Reviews subgraph];
+  gateway([Gateway]);
+  end
+  subgraph "Apollo cloud"
+  registry{{Apollo Schema Registry}};
+  uplink{{Apollo Uplink}}
+  end
+  serviceA & serviceB -->|Pushes schema| registry;
+  registry -->|Updates config| uplink;
+  gateway -->|Polls for config changes| uplink;
+  class registry secondary;
+  class uplink secondary;
+```
+
+To maximize uptime, Uplink is hosted simultaneously at _two_ endpoints, one in GCP and one in AWS:
+
+* GCP: `https://uplink.api.apollographql.com/`
+* AWS: `https://aws.uplink.api.apollographql.com/`
+
+By default in versions of `@apollo/gateway` v0.45.0 and later, your gateway polls the GCP URL at a regular interval. Whenever a poll request fails, the gateway retries that request with the AWS URL. It continues swapping between the two URLs until the request succeeds, or until reaching the maximum number of retries.
+
+Even if a poll request fails all retries, the gateway continues polling at the next interval, and it continues using its most recently obtained configuration.
+
+> Verisons of `@apollo/gateway` prior to v0.45.0 don't support multiple Uplink endpoints and only use the GCP URL by default.
+
+## Configuring polling behavior
+
+You can configure the list of Uplink URLs your gateway uses, along with the number of retries it performs for a failed poll request.
+
+### Uplink URLs
+
+You can provide a custom list of URLs for the gateway to use when polling Uplink. For example, you can prioritize the AWS URL or omit one of the URLs entirely.
```

Comment:
I find the use cases listed to be the only ones I can think of and also not very compelling. We might nudge users away from this option or suggest that it's unlikely they need to configure this.